Hero.py

```diff
@@ -0,0 +1,65 @@
+
+class hero:
+    extra_hp = 0.05
+    extra_damage = 0.05
+    enough_coin = 2
+    def __init__(self,name_of_hero)->None:
+        self._name_of_hero = name_of_hero
+        self._max_hp = 10
+        self._hp = 10
+        self._damage = 2
+        self._level = 1
+        self._coins = 0
+        self._defended = False
+    def __str__(self) -> str:
+        return f'name of here:{self._name_of_hero} hp{self._hp} damage:{self._damage} level:{self._level} coins:{self._coins}'
+
+    @property
+    def damage(self):
+        return self._damage
+    @property
+    def hp(self):
+        return self._hp
+
+    def change_defended(self):
+        if self._defended:
+            self._defended = False
+        else:
+            self._defended = True
```

Comment:
you could write it `param =! param`.             
if he is defended, you are not supposed to cancel it